tbx/work/models.py

```diff
@@ -143,12 +157,131 @@ def type(self):
     ]
 
 
+class WorkPage(SocialFields, Page):
+    template = "patterns/pages/work/work_page.html"
+    parent_page_types = ["WorkIndexPage"]
+
+    intro = RichTextField(blank=True)
+    client = models.CharField(max_length=255, blank=True)
+    date = models.DateField("Post date", blank=True, null=True)
+    body_word_count = models.PositiveIntegerField(null=True, editable=False)
+
+    header_image = models.ForeignKey(
+        "images.CustomImage",
+        verbose_name="Image",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
+    header_caption = models.CharField("Caption", max_length=255, blank=True)
+    header_attribution = models.CharField("Attribution", max_length=255, blank=True)
+
+    body = StreamField(WorkStoryBlock(), use_json_field=True)
+
+    listing_summary = models.CharField(max_length=255, blank=True)
+    related_services = ParentalManyToManyField(
+        "taxonomy.Service", related_name="case_studies"
+    )
+
+    content_panels = Page.content_panels + [
+        FieldPanel("intro"),
+        InlinePanel("authors", label="Author", min_num=1),
+        FieldPanel("client", classname="client"),
+        FieldPanel("date"),
+        MultiFieldPanel(
+            [
+                FieldPanel("header_image"),
+                FieldPanel("header_caption"),
+                FieldPanel("header_attribution"),
+            ],
+            heading="Header image",
+        ),
+        FieldPanel("body"),
+    ]
+
+    promote_panels = [
+        MultiFieldPanel(Page.promote_panels, "Common page configuration"),
+        FieldPanel("listing_summary"),
+        FieldPanel("related_services", widget=forms.CheckboxSelectMultiple),
+        MultiFieldPanel(SocialFields.promote_panels, "Social fields"),
+    ]
+
+    @property
+    def type(self):
+        return "CASE STUDY"
+
+    @cached_property
+    def services(self):
+        return self.related_services.all()
+
+    @property
```

Comment:
Let's make this value reusable once the query has been made:

```suggestion
    @cached_property
```

Reply:
done in 44f8861